docs/4.3. Controller to Authorization Server Interactions.md

```diff
@@ -278,4 +278,4 @@ Content-Type: application/json
 }
 ```
 
-Note that if a Controller is unable to contact an Authorization Server, it should attempt to contact another Authorization Server from the discovered list until this [either succeeds or the list is exhausted](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys).
+Note that if a Controller is unable to contact an Authorization Server, it ought to attempt contacting another Authorization Server from the discovered list until this [either succeeds or the list is exhausted](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys).
```

Comment:
This ought to be made consistent with the change to 4.1. in the other PR, I think?

Reply:
This is different to 4.1 in that here the Controller does not need to fetch the public keys (as it does not act as a Resource Server) and therefore doesn't do public key management. It's only the strategy for contacting the Auth Server that applies.

Reply:
Modified PR